docs/contributing.md

```diff
@@ -1,7 +1,12 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+We'd love to accept your patches and contributions to this project. A good
+starting step to get familiar with the project and set up a development
+enviroment as per [demo](docs/demo/demo.md). After running through the demo, a
+good second step is to pick up an open
+[issue](https://github.com/google/ml-compiler-opt/issues) or create one that you
+would like to work on and submit a patch for. There are also just a few small
+guidelines you need to follow.
```

Comment:
nit: reference somehow the guidelines; (I also don't know what you're referring to here :) ). Do you mean "run yapf"?

Reply:
I meant all the guidlines that follow (like license, code format, review process). Updated the sentence to be more precise.